src/plugins/data/public/search/collectors/create_usage_collector.ts

```diff
@@ -20,22 +21,27 @@ export const createUsageCollector = (
     return application.currentAppId$.pipe(first()).toPromise();
   };
 
-  return {
-    trackQueryTimedOut: async () => {
-      const currentApp = await getCurrentApp();
-      return usageCollection?.reportUiCounter(
-        currentApp!,
-        METRIC_TYPE.LOADED,
-        SEARCH_EVENT_TYPE.QUERY_TIMED_OUT
-      );
-    },
-    trackQueriesCancelled: async () => {
+  const getCollector = (metricType: UiCounterMetricType, eventType: SEARCH_EVENT_TYPE) => {
+    return async () => {
       const currentApp = await getCurrentApp();
-      return usageCollection?.reportUiCounter(
-        currentApp!,
-        METRIC_TYPE.LOADED,
-        SEARCH_EVENT_TYPE.QUERIES_CANCELLED
-      );
-    },
+      return usageCollection?.reportUiCounter(currentApp!, metricType, eventType);
+    };
+  };
+
+  return {
+    trackQueryTimedOut: getCollector(METRIC_TYPE.LOADED, SEARCH_EVENT_TYPE.QUERY_TIMED_OUT),
+    trackQueriesCancelled: getCollector(METRIC_TYPE.LOADED, SEARCH_EVENT_TYPE.QUERIES_CANCELLED),
+    trackSessionSentToBackground: getCollector(
+      METRIC_TYPE.CLICK,
+      SEARCH_EVENT_TYPE.SESSION_SENT_TO_BACKGROUND
+    ),
+    trackSessionSavedResults: getCollector(
+      METRIC_TYPE.CLICK,
+      SEARCH_EVENT_TYPE.SESSION_SAVED_RESULTS
+    ),
```

Comment:
Track navigating to management from session indicator

Reply:
Track visits to session management panel

Reply:
I'm not seeing a track of visits to the management page. 
Is it here?

Reply:
Yes, it's `trackViewSessionsList`